src/java/main/org/apache/zookeeper/Version.java

```diff
@@ -20,7 +20,7 @@
 
 public class Version implements org.apache.zookeeper.version.Info {
 
-    public static int getRevision() {
+    public static String getRevision() {
```

Comment:
@rakeshadr My Google searches didn't bring anything about external  ZK revision number use. Of course, many in-house systems can rely on it. So, I am clueless about the best strategy here. :(

It would be quite annoying to break internal build tools, but this is an unfortunate side effect of a fundamental infra change (VCS). :(

@fpj wdyt?